### PR TITLE
Oleh pavlyk

### PR DESCRIPTION
**Pull request by Slaiit**
newest pull request

### CODE REVIEW

File: BilousS/HW10/Practical task(outside)/Building Spheres.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from math import pi
 
+
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Building Spheres.py:3]: good job

File: MakariiHlyva/HW10/CodeWars/main.py [hunk 2]
```diff
@@ -3,22 +3,22 @@
 # class Ball(object):
 #     def __init__(self, ball_type = 'regular'):
 #         self.ball_type = ball_type
-
+#
```
Comment (kolyasalubov) [inline MakariiHlyva/HW10/CodeWars/main.py:6]: good job

File: OlehPavlyk/HW10/CodeWars/1_Regular Ball Super Ball/main.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class Ball(object):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW10/CodeWars/1_Regular Ball Super Ball/main.py:1]: good job

File: OlehPavlyk/HW10/CodeWars/2_ Color Ghost/main.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from random import choice
```
Comment (kolyasalubov) [inline OlehPavlyk/HW10/CodeWars/2_ Color Ghost/main.py:1]: good job

File: OlehPavlyk/HW10/CodeWars/3_Basic subclasses - Adam and Eve/main.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+class Human:
```
Comment (kolyasalubov) [inline OlehPavlyk/HW10/CodeWars/3_Basic subclasses - Adam and Eve/main.py:1]: good job

File: OlehPavlyk/HW10/CodeWars/4_Classy Classes/main.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+class Person:
```
Comment (kolyasalubov) [inline OlehPavlyk/HW10/CodeWars/4_Classy Classes/main.py:1]: good job

File: OlehPavlyk/HW10/CodeWars/5_Building Spheres/main.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
+from math import pi
```
Comment (kolyasalubov) [inline OlehPavlyk/HW10/CodeWars/5_Building Spheres/main.py:1]: good job

File: OlehPavlyk/HW10/CodeWars/6_Python's Dynamic Classes/main.py [hunk 8]
```diff
@@ -0,0 +1,3 @@
+def class_name_changer(cls, new_name):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW10/CodeWars/6_Python's Dynamic Classes/main.py:1]: good job

File: OlehPavlyk/HW5/PT1/main.py [hunk 9]
```diff
@@ -0,0 +1,7 @@
+lst = [1, 3, 2, 4, 6, 5, 7, 8, 9]
```
Comment (kolyasalubov) [inline OlehPavlyk/HW5/PT1/main.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/10_Convert boolean values to strings 'Yes' or 'No'..py [hunk 10]
```diff
@@ -0,0 +1,6 @@
+def bool_to_word(boolean):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/10_Convert boolean values to strings 'Yes' or 'No'..py:1]: good job

File: OlehPavlyk/HW7/CodeWars/11_Counting sheep....py [hunk 11]
```diff
@@ -0,0 +1,6 @@
+def count_sheeps(sheep):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/11_Counting sheep....py:1]: good job

File: OlehPavlyk/HW7/CodeWars/12_Is this my tail.py [hunk 12]
```diff
@@ -0,0 +1,6 @@
+def correct_tail(body, tail):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/12_Is this my tail.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/1_Jenny's secret message.py [hunk 13]
```diff
@@ -0,0 +1,6 @@
+def greet(name):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/1_Jenny's secret message.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/2_Simple Find The Distance Between Two Points.py [hunk 14]
```diff
@@ -0,0 +1,3 @@
+def distance(x1, y1, x2, y2):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/2_Simple Find The Distance Between Two Points.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/3_No yelling!.py [hunk 15]
```diff
@@ -0,0 +1,3 @@
+def filter_words(st):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/3_No yelling!.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/4_Convert a Number to a String!.py [hunk 16]
```diff
@@ -0,0 +1,2 @@
+def number_to_string(num):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/4_Convert a Number to a String!.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/5_Reversing Words in a String.py [hunk 17]
```diff
@@ -0,0 +1,7 @@
+def reverse(st):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/5_Reversing Words in a String.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/6_Reverse List Order.py [hunk 18]
```diff
@@ -0,0 +1,5 @@
+def reverse_list(l):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/6_Reverse List Order.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/7_ Multiples of 3 or 5.py [hunk 19]
```diff
@@ -0,0 +1,10 @@
+def solution(number):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/7_ Multiples of 3 or 5.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/8_Will you make it.py [hunk 20]
```diff
@@ -0,0 +1,5 @@
+def zero_fuel(distance_to_pump, mpg, fuel_left):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/8_Will you make it.py:1]: good job

File: OlehPavlyk/HW7/CodeWars/9_Are You Playing Banjo.py [hunk 21]
```diff
@@ -0,0 +1,6 @@
+def are_you_playing_banjo(name):
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/CodeWars/9_Are You Playing Banjo.py:1]: good job

File: OlehPavlyk/HW7/PT3/main.py [hunk 22]
```diff
@@ -0,0 +1,27 @@
+# when upper and lower case counting separately
```
Comment (kolyasalubov) [inline OlehPavlyk/HW7/PT3/main.py:1]: good job

File: OlehPavlyk/HW8/PT1/main.py [hunk 23]
```diff
@@ -0,0 +1,4 @@
+from utils import *
```
Comment (kolyasalubov) [inline OlehPavlyk/HW8/PT1/main.py:1]: good job

File: OlehPavlyk/HW8/PT3/calculator.py [hunk 24]
```diff
@@ -0,0 +1,13 @@
+from math import pi
```
Comment (kolyasalubov) [inline OlehPavlyk/HW8/PT3/calculator.py:1]: good job

File: OlehPavlyk/HW8/PT3/main.py [hunk 25]
```diff
@@ -0,0 +1,27 @@
+import calculator
```
Comment (kolyasalubov) [inline OlehPavlyk/HW8/PT3/main.py:1]: good job

File: BilousS/HW10/Practical task(outside)/Building Spheres.py (hunk 1 above)
Comment (Slaiit) [reply BilousS/HW10/Practical task(outside)/Building Spheres.py:3]: thank you

**Pull request merged**